docs/index.asciidoc

```diff
@@ -23,7 +23,10 @@ include::{include_path}/plugin_header.asciidoc[]
 
 Send events over UDP
 
-Keep in mind that UDP will lose messages.
+Keep in mind that UDP is a lossy protocol, upon failure when writing to the UDP socket, the error will be logged and
```

Comment:
I think we can set better expectations here, using the language of RFC-768 itself:

> ~~~ asciidoc
> Keep in mind that UDP does not provide delivery or duplicate protection guarantees;
> even when this plugin succeeds at writing to the UDP socket, there is no guarantee that
> the recipient will receive exactly one copy of the event.
>
> When this plugin fails to write to the UDP socket, by default the event will be dropped
> and the error message will be logged. The <<plugins-{type}s-{plugin}-retry_count>> option
> in conjunction with the <<plugins-{type}s-{plugin}-retry_backoff_ms>> option can be used
> to retry a failed write for a number of times before dropping the event.
> ~~~

Reply:
great suggestion!

Reply:
pushed your suggestion. thanks.